misc/hooks/pre-commit-clang-format

```diff
@@ -140,6 +140,10 @@ do
     if grep -q "\-so_wrap." <<< $file; then
         continue;
     fi
+    if grep -q "tests/python_build" <<< $file; then
+          continue;
+    fi
+
```

Comment:
Extra empty line, style checks will complain about it :)

Reply:
Fixed. I wasn't aware there was a shell style checker in here too...